src/os/bluestore/NVMEDevice.cc

```diff
@@ -865,6 +865,7 @@ int NVMEDevice::open(const string& p)
   driver->register_device(this);
   block_size = driver->get_block_size();
   size = driver->get_size();
+  name = serial_number;
```

Comment:
huh, yeah dunno about that spdk change.  in any case, though, this is how we're identifying the device.  @yuyuyu101 is there something better to do here?